ci-operator/step-registry/ipi/conf/capability/additonalcaps/ipi-conf-capability-additonalcaps-commands.sh

```diff
@@ -98,6 +98,13 @@ if [[ "${selected_capability}" == "OperatorLifecycleManager" ]]; then
     enabled_capabilities=${enabled_capabilities/"marketplace"}
 fi
 
+# For OCP 4.15, CCO is required for non-BareMetal platforms
+if [[ "$ocp_major_version" == "4" ]] && [[ "$ocp_minor_version" == "15" ]] && [[ "${selected_capability}" == "CloudCredential" ]]; then
```

Comment:
```suggestion
if [[ "${selected_capability}" == "CloudCredential" ]]; then
```
On the line 66, only 4.15 has the possibility to select `CloudCredential`, so I thin do not need to check version.

Reply:
@pamoedom @aleskandro Do you know how to judge if the job is running on baremetal platform? Or shall we ignore the platform checking for now, because there is no baseline=none baremetal install yet?

Reply:
@jianlinliu I'm wondering if I could read it from install-config.platform. And allow CCO to be disabled if and only if platform = baremetal or none.

Reply:
@jianlinliu @aleskandro 
I guess I should read install-config.platform and allow CCO to be disabled only if platform is `baremetal` or `none`, right ?

Reply:
@aleskandro is it possible to use `CLUSTER_TYPE` to do that?

Reply:
@jianlinliu `CLUSTER_TYPE` appears to take many possible values. It is possible to use it (with some regex matches for example) but I guess it's not going to be straightforward. 
https://github.com/openshift/ci-tools/blob/21cd5e74813c87fa5e72e6f95aa0407b12014918/pkg/api/types.go#L1322

Reply:
Yeah, because we only focus on baremetal ones, we do not have to match all the supported cluster type via regex. I saw some other step is using something like the following to match baremetal cluster profile,
```
"${CLUSTER_TYPE}" =~ ^packet.*$|^equinix.*$
```
I am not if we some more other baremetal cluster types.

Reply:
The current cluster profile name has equinix in its name. I do not remember if CLUSTER TYPE comes from that and I'm afk today. I wonder if vsphere and other similar scenarios should be excluded too though

Reply:
For 4.15 the disablement of CCO is supported for BM only. 
Whether/when to support other platforms is still under discussion.